app/views/encode_records/index.html.erb

```diff
@@ -28,6 +28,9 @@ $(document).ready( function () {
     stateSave: true,
     processing: true,
     serverSide: true,
+    order: [[ 6, 'desc' ]],
+    lengthMenu: [ [10, 20, 50, -1], [10, 20, 50, "All"] ],
```

Comment:
Do we really want an "All" option?  I'm just afraid of the impact on server/DB/client.  Or maybe it won't be that bad because it will probably take a long time to get to 100,000+.

Reply:
Good point. I didn't really consider the implications; I just copied from an example.